### DISCUSSION TITLE
How to Improve the Detection Accuracy of This Character?

### DISCUSSION THREAD

**Discussion by MAbdElRaouf** [ocrmypdf/OCRmyPDF#1273] [Q&A]
Hello,

I have been using this great library to perform OCR on invoices and it has been working great so far.

I'm having difficulty trying to get OCRmyPDF to recognize the '**9**' in this picture (part of an invoice) correctly:

![Screenshot 2024-03-09 215658](https://github.com/ocrmypdf/OCRmyPDF/assets/60540684/7cb0b02a-4c89-4716-bbb2-478649bb701e)

I have tried different `oversample` values (None, 200, 350, 600, 800, 1000) combined with different `tesseract_pagesegmode` (1, 4, 6, 11, 12), and the generated text would always recognize the '**9**' as $, €, 6, or omit it altogether.

What can be done to improve the detection accuracy in this case and similar cases?

Thank you.

**Comment by jbarlow83** [comment]
It may be possible to fine-tune Tesseract to improve its recognition with some labeled examples, although it may increase errors in other cases. OCRmyPDF doesn't have anything on its own to help with this.
